src/popup.html

```diff
@@ -52,7 +52,10 @@
             <div class="messages" id="messages">
             </div>
         </div>
-        <span style="z-index: 1;" class="scroll-icon" id="scrollIcon">
+        <span class="scroll-icon-top" id="scrollIconTop">
+            <svg viewBox="0 0 24 24" style="display: inline-block; color: rgb(255, 255, 255); fill: rgb(255, 255, 255); height: 30px; width: 24px; user-select: none; transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms; line-height: 40px;"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"></path></svg>
+        </span>
+        <span style="z-index: 1;" class="scroll-icon-bottom" id="scrollIconBottom">
         <svg viewBox="0 0 24 24" style="display: inline-block; color: rgb(255, 255, 255); fill: rgb(255, 255, 255); height: 30px; width: 24px; user-select: none; transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms; line-height: 40px;"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"></path></svg>
```

Comment:
Just align this as well. Scroll icon looks good to me now.

Reply:
@rupav done

Reply:
I meant to say align the `svg` tag by 1 tab right, as it's inside the enclosing span tag.

Reply:
@fragm3 Can you make changes suggested by @rupav so that I can go ahead with the merge